apps/server/src/apps/helpers/run-bootstrap-command.ts

```diff
@@ -0,0 +1,28 @@
+import { DomainErrorHandler } from '@core/error';
+import { BootstrapConsole } from 'nestjs-console';
+
+export const runBootstrapCommand = async <T>(module: T): Promise<void> => {
```

Comment:
I think this should be called `runConsoleApp`. It does not only bootstrap the app.

Otherwise it looks like a good abstraction to me.